epsagon_kubernetes_exporters_deploy.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+function is_positive_answer {
+    answer=$1
+    if [ ${answer} == 'y' ] ; then
+        answer='Y'
+    fi
+    if [ ${answer} == 'Y' ] ; then
+        return 0;
+    fi
+    return 1;
+}
+
+function install_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    EXPORTER_FILENAME=$3
+    KUBECTL="kubectl --context ${CONTEXT} --kubeconfig=${CONFIG}"
+    ${KUBECTL} apply -f ${EXPORTER_FILENAME}
+    rm -f ${EXPORTER_FILENAME}
+}
+
+function apply_celery_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    echo 'Please insert the messaging broker url used by your Celery, for example:'
+    echo "rabbitmq: amqp://guest:guest@rabbitmq-service.monitoring:5672//"
```

Comment:
Let's provide a celery example here instead of rabbitmq / redis

Reply:
its actually a message queue service the celery is using  (I mean, we dont install any rabbitmq/redis exporter, celery just need to use any of those)